docs/docs/how-to/sourcing-data/headless-cms.md

```diff
@@ -53,6 +53,7 @@ The following CMSs have high popularity among Gatsby users and support key funct
 | [Builder.io](https://www.builder.io/)     | [guide](/docs/sourcing-from-builder-io/)                                    | [docs](/plugins/@builder.io/gatsby/)                |                                                             |
 | [Flotiq](https://flotiq.com/)             | [guide](/docs/sourcing-from-flotiq/)                                        | [docs](/plugins/gatsby-source-flotiq)               |
 | [Webiny](https://www.webiny.com/)         | [guide](https://www.webiny.com/docs/headless-cms/integrations/gatsby)       |                                                     | [starter](/starters/webiny/gatsby-starter-webiny/)          |
+| [Dialoguewise](https://dialoguewise.com/) | [guide](/docs/sourcing-from-dialoguewise)                                   | [docs](/plugins/gatsby-source-dialoguewise)         |                                                             |
```

Comment:
Please link to docs on your own website, we don't want to add any more docs to our own repository

Reply:
Thanks @LekoArts for getting back on this. I've amended the commit accordingly. Appreciate your help on this.